CONTRIBUTING.md

```diff
@@ -150,14 +150,24 @@ See our separate [style guide](STYLE.md) document.
     to indicate their reach.
   - We publish detailed [release notes](https://www.cockroachlabs.com/docs/releases/)
     describing most non-test changes. To facilitate this, at least one commit in every
-    PR (possibly the PR message/merge commit) should contain a brief description of
+    PR (preferrably the PR message/merge commit) should contain a brief description of
     the change in terms that a user would recognize. This description should be prefixed
-    with "Release note:". For example, a commit like ["distsql: pre-reserve memory needed
+    with "Release note (category):", where the "category" is one of the following:
+    - cli change
+    - sql change
+    - ui change
```

Comment:
Can you call this "admin UI" just so its unambiguous? (I think of SQL and the CLI as user interfaces too.)

Reply:
Or "web UI", since admins use the CLI too.

Reply:
Done.